manifests/init.pp

```diff
@@ -3,85 +3,43 @@
 # Manage facter
 #
 class facter (
-  $manage_package         = undef,
-  $package_name           = 'facter',
-  $package_ensure         = 'present',
-  $manage_facts_d_dir     = true,
-  $purge_facts_d          = false,
-  $facts_d_dir            = '/etc/facter/facts.d',
-  $facts_d_owner          = 'root',
-  $facts_d_group          = 'root',
-  $facts_d_mode           = '0755',
-  $path_to_facter         = '/usr/bin/facter',
-  $path_to_facter_symlink = '/usr/local/bin/facter',
-  $ensure_facter_symlink  = false,
-  $facts_hash             = {},
-  $facts_hash_hiera_merge = false,
-  $facts_file             = 'facts.txt',
-  $facts_file_owner       = 'root',
-  $facts_file_group       = 'root',
-  $facts_file_mode        = '0644',
+  Boolean                    $manage_facts_d_dir     = true,
```

Comment:
just one space between things like

`DataType $variable = 'value',`

With the arrival of data types, keeping things in columns doesn't make sense anymore as the columns can get WAY WAY spaced out and when you add one it causes a bunch of code churn to respace things.